plugins/jobs/web_client/views/CheckBoxMenu.js

```diff
@@ -43,17 +43,20 @@ var CheckBoxMenu = View.extend({
         }));
         this._renderContent();
     },
+
     setItems: function (items) {
         this.items = items;
         this._renderContent();
     },
-    allItemChecked: function () {
-        return _.every(_.values(this.items));
+
+    allItemsChecked: function () {
```

Comment:
No, this isn't released. I agree, I don't think we need a public method for this.